dashboard/utils/configuration_tools.lua

```diff
@@ -45,7 +45,11 @@ function M.resolve_password(task, entry)
         if entry.command then
             local exit_code, stdout, stderr = task:execute(entry.command, entry.arguments)
             if exit_code == 0 then
-                return (stdout:match '^[^\n]*')
+                if entry.multiline ~= nil and entry.multiline then
```

Comment:
added a nil check here because there's no schema requiring it (yet?)

Reply:
Oh, I see. Maybe just:    entry.multiline == true, then?